config/scheduler/settings.ini

```diff
@@ -48,6 +48,9 @@ pa.scheduler.core.nodepingfrequency=20
 # Also used by a node to retry to send the result of a task to the scheduler
 pa.scheduler.core.node.ping.attempts=1
 
+# Time in milliseconds before exiting the scheduling process
```

Comment:
when the scheduler receives a kill() request (see org.ow2.proactive.scheduler.common.Scheduler.kill)